examples/gcs-static-website-example/README.md

```diff
@@ -0,0 +1,50 @@
+# Sample web app using GCS and GCE
+
+This code deploys a sample Web UI in a GCS bucket and a sample nodeJS based backend as container on a MIG.
+
+A GCLB exposes frontend and backend under a joined domain, exposing the backend service via the /api path.
+
+## How to deploy
+
+If you are using a Service Account to deploy the Terraform script, you can add the following line to the google provider in the providers.tf file:
+
+`impersonate_service_account = "SA_EMAIL"`
+
+Terraform will automatically impersonate this service account when deploying the infrastructure. The account which gcloud is currently authenticated with needs to have the "Service Account Token Creator" role on the service account. You can find out which account is currently authenticated with gcloud running `gcloud auth list`. If you are not logged in yet, run `gcloud auth login` to login.
+
+Keep in mind that your currently authenticated user still needs permission to access the state bucket, if you are using one.
+
+## Deploy the infrastructure via Terraform
+
+Run `terraform init` and `terraform apply`. 
+
+## Create the A record for your domain
+
+From the outputs, copy the IP address of the newly created load balancer. Register a new A record with yoru domain provider. The domain must be an exact match to the one you set as variable before running the script.
+
+## Deploy the backend application
+
+Authenticate against Artifact Registry:
+
+`gcloud auth configure-docker REGION.pkg.dev`
```

Comment:
based off of https://cloud.google.com/artifact-registry/docs/docker/authentication#:~:text=run%20the%20command%3A-,gcloud,-auth%20configure%2Ddocker

The line should read 
```suggestion
`gcloud auth configure-docker REGION-docker.pkg.dev`
```